### ISSUE TITLE
oil

### ISSUE THREAD

**Issue by mrelg** [falk-hueffner/metric-cooking#10]
Please add conversion to [g] for oils
not everyone likes to measure all liquids by [ml]
it can be messy

**Comment by falk-hueffner** [comment]
You are right, this would be useful sometimes. I wouldn't want to have it as default though, so it would need a preference panel, which I'm not sure how to do.
